catalogs/catalog-jdbc-doris/build.gradle.kts

```diff
@@ -61,7 +65,11 @@ tasks {
   }
   val copyCatalogLibs by registering(Copy::class) {
     dependsOn("jar", "runtimeJars")
-    from("build/libs")
+    from("build/libs") {
+      exclude("guava-*.jar")
+      exclude("log4j-*.jar")
+      exclude("slf4j-*.jar")
+    }
```

Comment:
Why do we need to remove these jars explicitly? If we're using them only in compile, maybe we can change from "implementation" to "compileOnly"?

Reply:
Those jars already exist in the server class path, duplicated jars are useless.

Reply:
>  If we're using them only in compile, maybe we can change from "implementation" to "compileOnly"?

Yeah, let me have a try.

Reply:
> > If we're using them only in compile, maybe we can change from "implementation" to "compileOnly"?
> 
> Yeah, let me have a try.

It seems that making it `compileOnly` can't pass tests and runs into classes not found problem.